### PR TITLE
Code quality enhancements

### PR DESCRIPTION
**Pull request by tusharmakkar08**
Fixed PEP8 issues and DRY'ed the code

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ beautifulsoup4==4.4.1
 python-magic==0.4.10
 requests==2.8.1
 wheel==0.24.0
+docopt==0.6.2
```
Comment (prodicus) [inline requirements.txt]: we moved on to the standard library instead of this

File: xkcd_dl/annotated.py [hunk 2]
```diff
@@ -23,55 +23,64 @@
 import requests
 import json
 import os
-from os.path import expanduser, join
-from os import getcwd
```
Comment (prodicus) [inline xkcd_dl/annotated.py]: Reasons for removal of line 27 and the `join` import?

File: xkcd_dl/annotated.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
```
Comment (prodicus) [inline xkcd_dl/annotated.py]: As discussed in #22 . This file is just for reference. The main endpoint would be the `cli.py` file

File: xkcd_dl/cli.py [hunk 4]
```diff
@@ -13,8 +13,7 @@
 import shutil
 import json
 import os
-from os.path import expanduser, join
-from os import getcwd
```
Comment (prodicus) [inline xkcd_dl/cli.py]: reasons for removal of `getcwd` and `join`?

File: xkcd_dl/cli.py [hunk 5]
```diff
@@ -236,23 +228,22 @@ def set_custom_path(custom_path):
...
     return path_was_set
 
+
 def show_xkcd(num):
     download_one(read_dict(), num)
-    path = '{current_directory}/xkcd_archive/{name}/'.format(
-        current_directory=WORKING_DIRECTORY, 
-        name=num
-    )
+    path = '{current_directory}/xkcd_archive/{name}/'.format(current_directory=WORKING_DIRECTORY, name=num)
```
Comment (prodicus) [inline xkcd_dl/cli.py]: this actually reduces readability, don't you think

File: requirements.txt (hunk 1 above)
Comment (tusharmakkar08) [inline requirements.txt]: It was present in `annotated.py` that's why added it.

File: xkcd_dl/annotated.py (hunk 3 above)
Comment (tusharmakkar08) [inline xkcd_dl/annotated.py]: I also guessed so, since most of the logic is repeated here. Let's just remove this file. What do you say?

File: xkcd_dl/cli.py (hunk 4 above)
Comment (tusharmakkar08) [inline xkcd_dl/cli.py]: These imports aren't used anywhere..

File: xkcd_dl/cli.py (hunk 5 above)
Comment (tusharmakkar08) [inline xkcd_dl/cli.py]: This looks like so because of the side-by-side view. Pls look at line 234 [here](https://github.com/tusharmakkar08/xkcd-dl/blob/828c2f7e518adecd67dc91f2341c22d598c457d9/xkcd_dl/cli.py).. Can revert this change, if you still feel so. (following 120 char limit)

Comment (prodicus) [inline xkcd_dl/cli.py]: The word limit should not cross 79. But I am pretty sure that there are some parts in my code where this is not followed. 

I thought about keeping back the old `annoted.py` for historical purposes. 
Cheers!

Comment (tusharmakkar08) [inline xkcd_dl/cli.py]: Retracted the change and pushed the modified one. Let me know if you have any other concerns

File: requirements.txt [hunk 6]
```diff
@@ -2,3 +2,4 @@ beautifulsoup4==4.4.1
 python-magic==0.4.10
 requests==2.8.1
 wheel==0.24.0
```
Comment (prodicus) [inline requirements.txt]: The `wheel` and the `docopt` spec is not required.

And the `requests` module got upgraded to `2.10.*`. I am not sure on that. So check it out.

File: xkcd_dl/annotated.py (hunk 3 above)
Comment (prodicus) [inline xkcd_dl/annotated.py]: Please revert the changes made on `annoted.py`. As we have discussed, it is for reference purposes for other contributors.

**Comment by prodicus** [comment]
One suggestion for a faster code review @tusharmakkar08 is to break large commits into small, as we currently don't have any functional tests for `xkcd-dl` (@BlitzKraft  is on it he says)

So it comes down to some manual testing. So I would suggest you work on a smaller refactor than this. I hope you understand. Closing this PR
